apps/docs/content/components/CTABanner.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: CTABanner
+status: Experimental
+figma: 'https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/Primer-Brand?node-id=539%3A14285&t=RJEWZhVUtrRQ0pKh-0'
+source: https://github.com/primer/brand/tree/main/src/CTABanner/CTABanner.tsx
+storybook: '/brand/storybook/?path=/story/components-ctabanner--playground'
+description: Use CTABanner to draw attention to or create urgency around a user action.
+---
+
+```js
+import {CTABanner, Button} from '@primer/react-brand'
+import {Label} from '@primer/react'
```

Comment:
This needs to be outside the code snippet scope.. it's currently rendering in the docs..

![Screenshot 2022-12-07 at 15 12 21](https://user-images.githubusercontent.com/13340707/206216563-4317fbaa-4531-40f6-b95b-f5045507f48b.png)